pidtree_bcc/plugins/sourceipmap.py

```diff
@@ -0,0 +1,60 @@
+import os
+import staticconf
+from functools import partial
+from pidtree_bcc.plugin import BasePlugin
+
+def hosts_loader(filename):
+    return_dict = {}
+    with open(filename, 'r') as mapfile:
+        lines = [line.strip() for line in mapfile
+                 if not line.startswith('#') and line.strip() != '']
+    for line in lines:
+        splitline = line.split()
+        return_dict[splitline[0]] = " ".join(splitline[1:])
+    return return_dict
+
+def build_configuration(filename, namespace):
+    config_loader = partial(staticconf.loader.build_loader(hosts_loader),
+                            filename,
+                            namespace=namespace,
+                            flatten=True)
+    reloader = staticconf.config.ReloadCallbackChain(namespace)
+    return staticconf.config.ConfigurationWatcher(config_loader,
+                                                  filename,
+                                                  min_interval=2,
+                                                  reloader=reloader)
+
+        
+class Sourceipmap(BasePlugin):
+    """ Plugin for mapping source ip to a name """
+    def __init__(self, args):
+        self.validate_args(args)
```

Comment:
Sorry, I completely missed this in the previous review, it was supposed to be `super().__init__(args)`

Reply:
Yeah, I need to rethink this bit tbh so I've just directly called the validate for now.